### PR TITLE
add cc:tweaked

### PR DESCRIPTION
**Pull request by electron271**
may need some configuration? unsure

### CODE REVIEW

**Comment by electron271** [comment]
works client side, will check server side eventually

**Comment by RPMYT** [comment]
looks good to me but needs HTTP API enabled if that isn't default

File: index.toml [hunk 1]
```diff
@@ -2,7 +2,12 @@ hash-format = "sha256"
 
 [[files]]
 file = "config/yosbr/options.txt"
-hash = "292014c16b30631d5c70997da09b637deec20997c5d19952666e90e77c4b2bad"
+hash = "b3561d15b4cb62add9bf7211dd2b13c73762f156546602081b8185c0027f41aa"
+
+[[files]]
```
Comment (RPMYT) [inline index.toml:7]: might need to include config for HTTP

**Review by RPMYT — CHANGES_REQUESTED** [review]

File: index.toml (hunk 1 above)
Comment (electron271) [reply index.toml:7]: image
http appears to work (this is on the server, creative mode no op)

**Comment by electron271** [comment]
do not merge i am fixing the shasum

**Comment by electron271** [comment]
ready to merge

**Review by RPMYT — APPROVED** [review]
HTTP seems enabled by default, looks good to me!

**Pull request merged**